07.bowling_object/lib/bowling.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require_relative 'shot'
+require_relative 'frame'
+require_relative 'bonus'
+require_relative 'game'
+
+input = ARGV[0]
+game = Game.new(input)
+p game.score
```

Comment:
```suggestion
puts game.score
```

pは「主にデバッグ用」です。
https://rurema.clear-code.com/3.0.0/method/Kernel/m/p.html

Reply:
そうでした...putsに修正しました。